In [2]:
import pandas as pd
import numpy as np
import matplotlib as plt
from pathlib import Path


%matplotlib inline

In [6]:
data_dir = Path(".").absolute().parent

WindowsPath('C:/Users/benik/PycharmProjects/Plant Pathology')

In [7]:
df = pd.read_csv(data_dir/'data'/'train.csv')
df

image_id  healthy  multiple_diseases  rust  scab
0        Train_0        0                  0     0     1
1        Train_1        0                  1     0     0
2        Train_2        1                  0     0     0
3        Train_3        0                  0     1     0
4        Train_4        1                  0     0     0
...          ...      ...                ...   ...   ...
1816  Train_1816        0                  0     0     1
1817  Train_1817        1                  0     0     0
1818  Train_1818        1                  0     0     0
1819  Train_1819        0                  0     1     0
1820  Train_1820        0                  0     0     1

[1821 rows x 5 columns]

* Muiltiple does not count as both the other diseases.
* There are only 4 labels
* There are 1820 images for the Train segment.